control/init.sh

```diff
@@ -17,24 +17,24 @@ function realpath() {
 
 
 if [ -z ${CI_TEST} ]; then
+    if [[ $(cat /etc/os-release | grep '^VERSION=' | tr -dc '0-9.') == *18.04* ]] && hash python3.10 2>/dev/null; then
+        # if on 18.04 and 3.10 is installed, figure out which name to use for a 3.10 virtualenv
+        if [[ -f $VENV/bin/python3.6 ]]; then
+            # if a 3.6 environment with the $CCHQ_VIRTUALENV name already exists, append '-3.10'
+            CCHQ_VIRTUALENV=$CCHQ_VIRTUALENV-3.10
+            VENV=~/.virtualenvs/$CCHQ_VIRTUALENV
+        fi
```

Comment:
What do you think of always using `cchq-3.10` on 18.04? It simplifies the logic.
```suggestion
        # if on 18.04 and 3.10 is installed, cchq-3.10 is the default virtualenv name
        if [[ "$CCHQ_VIRTUALENV" == cchq ]]; then
            CCHQ_VIRTUALENV=cchq-3.10
            VENV=~/.virtualenvs/$CCHQ_VIRTUALENV
        fi
```

Reply:
See 38a57d19c00901b8085204834bff68134cf46e0f